<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Dee-PC" failures="1" tests="3" name="com.herokuapp.theinternet.alertstests.AlertsTests" time="37.815" errors="0" timestamp="13 Oct 2018 15:25:14 GMT" skipped="0">
  <testcase classname="com.herokuapp.theinternet.alertstests.AlertsTests" name="jsAlertTest" time="12.381"/>
  <testcase classname="com.herokuapp.theinternet.alertstests.AlertsTests" name="jsDismissTest" time="12.501"/>
  <testcase classname="com.herokuapp.theinternet.alertstests.AlertsTests" name="jsPromptTest" time="12.933">
    <failure message="The following asserts failed:
	Alert message is not expected. 
Should be &#039;I am a JS prompt&#039;, but it is &#039;I am a JS prompt[FAIL]&#039; expected [true] but found [false],
	result is not expected. 
Should be &#039;You entered: Hello Alert, its Dmitry here&#039;, but it is &#039;You entered: Hello Alert, it&#039;s Dmitry here[FAIL]&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Alert message is not expected. 
Should be 'I am a JS prompt', but it is 'I am a JS prompt[FAIL]' expected [true] but found [false],
result is not expected. 
Should be 'You entered: Hello Alert, its Dmitry here', but it is 'You entered: Hello Alert, it's Dmitry here[FAIL]' expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.herokuapp.theinternet.alertstests.AlertsTests.jsPromptTest(AlertsTests.java:113)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- jsPromptTest -->
</testsuite> <!-- com.herokuapp.theinternet.alertstests.AlertsTests -->
